### ISSUE TITLE
about gnb.model

### ISSUE THREAD

**Issue by zhouxinmin** [MashiMaroLjc/dudulu#1]
Could you tell me how to generate the gnb.model file ?

**Comment by dannyhou** [comment]
**Here is how the model saved:**
 
from sklearn.externals import joblib

feature, label = get_feature("train.txt", kw_list)
gnb = GaussianNB()
gnb = gnb.fit(feature, label)
joblib.dump(gnb, 'model/gnb.model')

**Comment by zhouxinmin** [comment]
oh，thank you！



 zhouxinmin
 
From: dannyhou
Date: 2017-02-23 09:02
To: MashiMaroLjc/dudulu
CC: zhouxinmin; Author
Subject: Re: [MashiMaroLjc/dudulu] about gnb.model (#1)
Here is how the model trained:
feature, label = get_feature("train.txt", kw_list)
gnb = GaussianNB()
gnb = gnb.fit(feature, label)
joblib.dump(gnb, 'model/gnb.model')

**Comment by zhouxinmin** [comment]
oh，thank you!
